en/docs/install-and-setup/deploying-wso2-api-manager/production-deployment-guidelines.md

```diff
@@ -1,18 +1,27 @@
 # Production Deployment Guidelines
 
-The requirements for deploying WSO2 products can change based on the deployment scenario and pattern. The recommendations in this topic are for general production use, assuming moderate load conditions. For situations where a high volume of traffic is expected and if there are large deployments, these guidelines may not be sufficient. See [Troubleshooting in Production Environments](https://docs.wso2.com/display/ADMIN44x/Troubleshooting+in+Production+Environments) for information on how to obtain and analyze information to solve production issues. The following are the topics addressed in this section.
+The requirements for deploying WSO2 products can change based on the deployment scenario and pattern. The 
+recommendations in this topic are for general production use, assuming moderate load conditions. For situations where
```

Comment:
recommendations **under** this topic

Reply:
This section was not changed by me. Only the formatting has been changed by the pr. Anyway fixed by 238139a755fcb1391f32be7af75e89190fd5c965